Wellthier/API/ExerciseAPIManager.m

```diff
@@ -51,4 +51,26 @@ - (void) fetchAllExercises: (void(^)(NSArray *exercises, NSError *error))complet
     [dataTask resume];
 }
 
+- (void) fetchBodyParts: (void(^)(NSArray *bodyParts, NSError *error))completion {
+    NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:[NSURL URLWithString:@"https://exercisedb.p.rapidapi.com/exercises/bodyPartList"]
+                                                           cachePolicy:NSURLRequestUseProtocolCachePolicy
+                                                       timeoutInterval:10.0];
+    [request setHTTPMethod:@"GET"];
+    [request setAllHTTPHeaderFields:self.headers];
+
+    NSURLSession *session = [NSURLSession sharedSession];
+    NSURLSessionDataTask *dataTask = [session dataTaskWithRequest:request
+                                                completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
+                                                    if (error) {
+                                                        NSLog(@"%@", error);
+                                                    } else {
+                                                        NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *) response;
+                                                        NSLog(@"%@", httpResponse);
+                                                        NSArray *dataDictionary = [NSJSONSerialization JSONObjectWithData:data options:NSJSONReadingMutableContainers error:nil];
+                                                        completion(dataDictionary, nil);
+                                                    }
+                                                }];
+    [dataTask resume];
```

Comment:
I see this implementation very much duplicate of the above method. Can we create seperate helper method to avoid duplication of creating NSURLSessionDataTask